controllers/cinder_controller.go

```diff
@@ -710,6 +750,28 @@ func (r *CinderReconciler) createHashOfInputHashes(
 	return hash, nil
 }
 
+func (r *CinderReconciler) transportURLCreateOrUpdate(instance *cinderv1beta1.Cinder) (*rabbitmqv1.TransportURL, controllerutil.OperationResult, error) {
+	transportURL := &rabbitmqv1.TransportURL{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-cinder-transport", instance.Name),
+			Namespace: instance.Namespace,
+		},
+	}
+
+	op, err := controllerutil.CreateOrUpdate(context.TODO(), r.Client, transportURL, func() error {
+		transportURL.Spec.RabbitmqClusterName = instance.Spec.RabbitMqClusterName
+
+		err := controllerutil.SetControllerReference(instance, transportURL, r.Scheme)
+		if err != nil {
+			return err
+		}
+
+		return nil
```

Comment:
nit: We could just `return err` in both cases

Reply:
Good point